### DISCUSSION TITLE
Fail to start a tensorboard session (Time out)

### DISCUSSION THREAD

**Discussion by JiaxinWang-thu** [microsoft/vscode-jupyter#9470] [General] [answered]
I come across this problem with info "We failed to start a TensorBoard session due to the following error: Error: Timed out after 60 seconds waiting for TensorBoard to launch." How can I deal with it? I need your help, thanks.

**Comment by rchiodo** [comment, marked as answer]
You should log an issue on the vscode-jupyter extension. We own the tensorboard integration. There's likely a problem with your tensorboard installation. 

Hopefully there should be more information in the 'Python' output tab from the launch of the tensorboard session.

**Reply by yongshangli** [reply]
I got the same notification.
![2022-03-24 160004](https://user-images.githubusercontent.com/59642489/159878090-eb57afa8-dcc8-4bd0-b36c-f657f70a5144.png)

**Reply by JiaxinWang-thu** [reply]
I reinstalled the related pakages, and the problem has been solved.

**Reply by mgrafu** [reply]
@JiaxinWang-thu can you share what you reinstalled? I'm facing the same issue and reinstalling tensorboard and extension seem not to be working
